django/db/backends/mysql/schema.py

```diff
@@ -33,6 +33,10 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     # removed during a "MODIFY" column statement.
     sql_delete_check = 'ALTER TABLE %(table)s DROP CONSTRAINT IF EXISTS %(name)s'
 
+    @property
+    def sql_column_default(self):
+        return ' DEFAULT (%s)' if self._supports_limited_data_type_defaults else ' DEFAULT %s'
```

Comment:
We should add parentheses only on MySQL > 8.0.13 and only for `_limited_data_types`. I don't think that a new class variable/property is a good solution. I would rather add a method
```python
def _column_default(self, field):
    return '%s'
```
that can be override in a MySQL backend, e.g.
```python
def _column_default(self, field):
    if (
        not self.connection.mysql_is_mariadb and
        self._supports_limited_data_type_defaults and
        self._is_limited_data_type(field)
    ):
        return '(%s)'
    return super()._column_default(field)
```

This can easily be reused in `_alter_column_default_sql()`.